lib/rpmsg/rpmsg_core.c

```diff
@@ -276,7 +276,7 @@ void rpmsg_rx_callback(struct virtqueue *vq)
 			/* Fatal error no endpoint for the given dst addr. */
 			return;
 
-		if (ept && ept->dest_addr == RPMSG_ADDR_ANY) {
+		if (ept && ept->dest_addr == RPMSG_ADDR_ANY && ept->addr != RPMSG_NS_EPT_ADDR) {
```

Comment:
@arnop2 I don't think this is needed anymore due to us always calling the cb now.  While I don't think it hurts, do we need this if check at all anymore?